### PR TITLE
React to recent ASP.NET changes

### PR DESCRIPTION
**Pull request by dougbu**
Packages aren't available on vnext feed yet but getting this out for discussion.
- also include some minor changes made in aspnet/aspnet.xunit fork

### CODE REVIEW

File: global.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {
-    "sdk": { "version": "1.0.0-rc1-16048" },
+    "sdk": { "version": "1.0.0-rc2-16090" },
```
Comment (dougbu) [inline global.json]: Not sure which version will reach vnext feed. Will update this when our CI succeeds.

File: src/xunit.runner.dnx/project.json [hunk 2]
```diff
@@ -29,10 +29,9 @@
         }
     },
     "dependencies": {
-        "Microsoft.Dnx.Testing.Abstractions": "1.0.0-rc1-*",
-        "Microsoft.Dnx.TestHost": "1.0.0-rc1-*",
-        "Microsoft.Extensions.PlatformAbstractions": "1.0.0-rc1-*",
-        "xunit.runner.utility": "2.1.0",
```
Comment (dougbu) [inline src/xunit.runner.dnx/project.json]: As mentioned in one of the commit descriptions, this dependency comes in transitively.

**Comment by dougbu** [comment]
:up::date: now have RC2 packages in the aspnetvnext feed. Also rebased my fork.

**Comment by bradwilson** [comment]
Thanks, as soon as RC1 final drops, we can get this merged for RC2 CI builds. :+1:

**Comment by dougbu** [comment]
RC1 final is now out e.g. https://www.nuget.org/packages/Microsoft.AspNet.Mvc/6.0.0-rc1-final

**Pull request merged**